bundles/org.openhab.binding.hive/pom.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
+  xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>org.openhab.addons.bundles</groupId>
+    <artifactId>org.openhab.addons.reactor.bundles</artifactId>
+    <version>2.5.5-SNAPSHOT</version>
+  </parent>
+
+  <artifactId>org.openhab.binding.hive</artifactId>
+
+  <name>openHAB Add-ons :: Bundles :: Hive Binding</name>
+
+  <properties>
+    <jacoco.version>0.8.5</jacoco.version>
+  </properties>
+
+  <dependencies>
+    <!-- Fluent assertions -->
+    <dependency>
+      <groupId>org.assertj</groupId>
+      <artifactId>assertj-core</artifactId>
+      <version>3.15.0</version>
+      <scope>test</scope>
+    </dependency>
+
+    <!-- Automated equals() and hashCode() testing -->
+    <dependency>
+      <groupId>nl.jqno.equalsverifier</groupId>
+      <artifactId>equalsverifier</artifactId>
+      <version>3.1.13</version>
+      <scope>test</scope>
+    </dependency>
+
+    <!-- Nicer junit4 parameterized tests -->
+    <dependency>
+      <groupId>pl.pragmatists</groupId>
+      <artifactId>JUnitParams</artifactId>
+      <version>1.1.1</version>
+      <scope>test</scope>
+    </dependency>
+  </dependencies>
+
+  <build>
```

Comment:
Remove the build and report sections. While they might be nice additions it's not used in any way here. Therefore it only in adds to the build time and maintenance. If such support would add value it should be added at another level in this project, not for each binding individually.